pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -296,11 +295,11 @@ func getPhaseInfoByPhaseType(worker *PipelineWorker, phaseType enterpriseApi.App
 }
 
 // updatePplnWorkerPhaseInfo updates the in-memory PhaseInfo(specifically status and retryCount)
-func updatePplnWorkerPhaseInfo(appDeployInfo *enterpriseApi.AppDeploymentInfo, retryCount int32, statusType enterpriseApi.AppPhaseStatusType) {
+func updatePplnWorkerPhaseInfo(appDeployInfo *enterpriseApi.AppDeploymentInfo, retryCount uint32, statusType enterpriseApi.AppPhaseStatusType) {
```

Comment:
I think its better to keep parity in the sense that either keep retryCount or tryCount.